plugins/bulk-import/src/utils/repository-utils.tsx

```diff
@@ -174,5 +168,39 @@ export const filterSelectedForActiveDrawer = (
 };
 
 export const urlHelper = (url: string) => {
+  if (!url || url === '') {
+    return url;
```

Comment:
if url is not there or empty we are returning the same value, should we return null/undefined or some value?

Reply:
returning '-'